### PR TITLE
Update a CHANGELOG and CMake versions.

### PR DESCRIPTION
**Pull request by mykhailo-kuchma**
Update a CHANGELOG and CMake versions according to the 0.7.0 release.

Signed-off-by: Mykhailo Kuchma <ext-mykhailo.kuchma@here.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,30 @@
+## v0.7.0-beta (04/09/2019)
+
+**Common**
+
+* New abstract TaskScheduler introduced to be used for async context
+  (default thread pool implementation provided).
+* Added UserAgent to mark all triggered requests.
+* China Lookup API URL added.
+* New HTTP abstraction layer introduced without singleton usage.
```
Comment (ksenteer) [inline CHANGELOG.md]: * Introduced new abstract TaskScheduler to use for async context; provided default thread pool implementation.
* Added UserAgent to mark all triggered requests.
* Added China Lookup API URL.
* Introduced new HTTP abstraction layer without singleton usage.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,30 @@
...
+
+**olp-edge-sdk-cpp-authentication**
+
+* Use new HTTP layer as Settings input parameter.
+* TaskScheduler now as Settings input parameter.
+* Removed raw std::thread() usage and switched to TaskScheduler for callbacks.
```
Comment (ksenteer) [inline CHANGELOG.md]: * Use new HTTP layer as an input parameter for Settings.
* Use TaskScheduler as an input parameter for Settings.
* Removed usage of raw `std::thread()` and switched to TaskScheduler for callbacks.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,30 @@
...
+
+**olp-edge-sdk-cpp-dataservice-read**
+
+* Use new HTTP layer as OlpClientSettings input parameter.
+* TaskScheduler now as OlpClientSettings input parameter.
+* Removed raw std::thread() usage and switched to TaskScheduler for callbacks.
+* Improved logging.
```
Comment (ksenteer) [inline CHANGELOG.md]: * Use TaskScheduler OlpClientSettings as an input parameter.
* Removed usage of raw `std::thread()` and switched to TaskScheduler for callbacks.
* Improved logging.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,33 @@
+## v0.7.0-beta (09/09/2019)
+
+**Common**
+
+* Introduced new abstract `olp::thread::TaskScheduler` to be used for async context; provided default thread pool implementation.
+* Added `UserAgent` HTTP header to mark all triggered requests.
+* China Lookup API URLs added (`here-cn` and `here-cn-dev` partitions).
```
Comment (ksenteer) [inline CHANGELOG.md]: Added China Lookup API URLs

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,33 @@
...
+* Introduced new HTTP abstraction layer `olp::http::Network` without singleton usage. Platform dependent implementations adapted and aligned with coding style.
+* Removed of old `olp::network::Network` with according implementations and helper classes.
```
Comment (ksenteer) [inline CHANGELOG.md]: What does `of` refer to? I suggest to remove it.

File: CHANGELOG.md (hunk 4 above)
Comment (mykhailo-kuchma) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 5 above)
Comment (mykhailo-kuchma) [reply CHANGELOG.md]: The line is changed to:
* `olp::network::Network` with according implementations and helper classes removed.

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,33 @@
...
+* Introduced new HTTP abstraction layer `olp::http::Network` without singleton usage. Platform dependent implementations adapted and aligned with coding style.
+* `olp::network::Network` with according implementations and helper classes removed.
+* Changed `olp::client::CancellationContext` to shared pimpl so we can remove the `std::shared_ptr` wrapper when using it across multiple requests.
+* Improved logging in core components.
+
+**olp-edge-sdk-cpp-authentication**
+
+* **Breaking Change** - Use new HTTP layer as input parameter for all requests. This is a mandatory parameter and users must provide it.
+* Use new `olp::thread::TaskScheduler`; provided as input parameter.
+* Removed usage of raw `std::thread` and switched to `olp::thread::TaskScheduler` for callbacks. If not provided callbacks are executed synchronously.
```
Comment (ksenteer) [inline CHANGELOG.md]: If unprovided callbacks executed synchronously.

Comment (andescu) [reply CHANGELOG.md]:
```suggestion
* Removed usage of raw `std::thread` and switched to `olp::thread::TaskScheduler` for callbacks. If unprovided, callbacks are executed synchronously.
```

To mark that unprovided belongs to TaskScheduler and not callbacks.

**Pull request merged**